sdk/monitor/opentelemetry-exporter-azuremonitor/opentelemetry/exporter/azuremonitor/_generated/_application_insights_client.py

```diff
@@ -15,12 +15,12 @@
     # pylint: disable=unused-import,ungrouped-imports
     from typing import Any
 
-from ._configuration import AzureMonitorExporterConfiguration
-from .operations import AzureMonitorExporterOperationsMixin
+from ._configuration import ApplicationInsightsClientConfiguration
+from .operations import ApplicationInsightsClientOperationsMixin
 from . import models
 
 
-class AzureMonitorExporter(AzureMonitorExporterOperationsMixin):
+class ApplicationInsightsClient(ApplicationInsightsClientOperationsMixin):
     """Opentelemetry Exporter for Azure Monitor.
```

Comment:
Fix comment.

Reply:
Nit: It looks like the comment is still not fixed? It it because it is autogenerated?

Reply:
This change is not there anymore in last commit, am I missing something?